cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -211,6 +211,45 @@ func main() {
 	log.Info().Msgf("Stopping osm-bootstrap %s; %s; %s", version.Version, version.GitCommit, version.BuildDate)
 }
 
+func (b *bootstrap) createDefaultMeshConfig() error {
+	// find presets config map to build the default MeshConfig from that
+	presetsConfigMap, err := b.kubeClient.CoreV1().ConfigMaps(b.namespace).Get(context.TODO(), presetMeshConfigName, metav1.GetOptions{})
+
+	// If the presets MeshConfig could not be loaded return the error
+	if err != nil {
+		return err
+	}
+
+	// Create a default meshConfig
+	defaultMeshConfig := buildDefaultMeshConfig(presetsConfigMap)
+	if _, err := b.meshConfigClient.ConfigV1alpha1().MeshConfigs(b.namespace).Create(context.TODO(), defaultMeshConfig, metav1.CreateOptions{}); err == nil {
+		log.Info().Msgf("MeshConfig (%s) created in namespace %s", meshConfigName, b.namespace)
+		return nil
+	}
+
+	if apierrors.IsAlreadyExists(err) {
+		return nil
```

Comment:
Could we add the logs that were present previously

Reply:
I just bubbled up the error to main() where there is one log line which wraps the error https://github.com/openservicemesh/osm/pull/4181/files#diff-2aa329ee12aeb0b3d3a60599559a0bbe07422779d9f730f4be4e668b275e06acR146

I thought a log line here might be confusing but I can add it in.

Reply:
added.